### ISSUE TITLE
Implement a function to deserialize NIP-01 event from `Span<felt252>`

### ISSUE THREAD

**Issue by maciejka** [keep-starknet-strange/joyboy#15]

**Comment by bhavyagosai** [comment]
hi can I take this one up??

**Comment by maciejka** [comment]
> hi can I take this one up??

Hey, nice to meet you. How long do you think it would take you?

**Comment by bhavyagosai** [comment]
shouldn't take long. What I essentially need is to implement a function which would deserialize the data received from the nip-01 event. Also afaik, this is blocked until #10 is resolved right where the struct will be created for this event?

**Comment by maciejka** [comment]
Right, but you do not need to wait until #10 is done to start the work.

**Comment by maciejka** [comment]
@bhavyagosai what is your telegram handle?

**Comment by maciejka** [comment]
@bhavyagosai I realized there is no need to deserialize events in Cairo. Please contact me on telegram (@aundumla) to discuss what you can work on.
